tests/model_tests/test_contact.py

```diff
@@ -1,7 +1,7 @@
 import uuid
 
 from core.db import db
-from core.db.entities import Contact, Organisation
+from core.db.entities import Contact, Organisation, Package
 
 
 def test_contact_organisation(app_ctx):
```

Comment:
Should this be renamed now? Does it need a docstring so the test is scoped a bit more clearly?

Reply:
yes and yes